src/core/MOM.F90

```diff
@@ -2152,6 +2167,13 @@ subroutine initialize_MOM(Time, Time_init, param_file, dirs, CS, restart_CSp, &
                  units="s", default=default_val, do_not_read=(dtbt > 0.0))
   endif
 
+  CS%dt_obc_seg_period = -1.0
+  call get_param(param_file, "MOM", "DT_OBC_SEG_UPDATE_OBGC", CS%dt_obc_seg_period, &
```

Comment:
As written, this new parameter is being logged in every MOM_parameter_doc.all file, including in many where it is meaningless.  Please add the argument `do_not_log=.not.associated(CS%OBC)` to this call or wrap it in on logical test for whether  open boundary conditions are actually being used.